Site/ASAB Maestro/Descriptors/influxdb.yaml

```diff
@@ -3,6 +3,12 @@ define:
   name: InfluxDB
   url: https://github.com/influxdata/influxdb
 
+secrets:
+  INFLUXDB_TOKEN: {}
```

Comment:
Why `{}`? What does it mean?

Don't you want to be specific?

Reply:
if it is empty, it still must be a dictionary, because I could not think about better solution right now and 64 bytes long token is default

Reply:
So don't relay on default ;-)